steps/start/benefit-type/content.en.json

```diff
@@ -20,17 +20,20 @@
     "sda": "Severe Disablement Allowance",
     "socialfund": "Social Fund",
     "uc": "Universal Credit (UC)",
-    "cc": "30 Hours Free Childcare Scheme",
+    "cc": "30 Hours Free Childcare",
     "childSupport": "Child Support",
     "bereavementsupportpaymentscheme": "Bereavement Support Payment Scheme",
-    "tc": "Tax Credits",
+    "tc": "Tax Credit",
```

Comment:
Should it be 'taxCredit' here?

Reply:
Yes according to the AC Then "Tax Credit is on the list of benefits that I can select (currently seen as Tax Credits)"